torch/csrc/utils/byte_order.cpp

```diff
@@ -462,7 +462,7 @@ void THP_encodeComplexFloatBuffer(
   }
 }
 
-void THP_encodeCompelxDoubleBuffer(
+void THP_encodeComplexDoubleBuffer(
```

Comment:
Is this never used? Should we just remove it?

Reply:
Seems it is not used

Reply:
I have some fixes in 2b62567